src/components/ChartView/ChartReadme.css

```diff
@@ -0,0 +1,4 @@
+.ChartReadme pre {
+  overflow: scroll;
```

Comment:
We should use `overflow: auto` so that scroll bar's are added by the browser only if needed.